core/org.eclipse.cdt.ui/META-INF/MANIFEST.MF

```diff
@@ -125,7 +125,8 @@ Require-Bundle: org.eclipse.cdt.core;bundle-version="[8.0.0,9.0.0)",
  org.eclipse.ui.views;bundle-version="[3.11.300,4.0.0)",
  org.eclipse.ui.workbench.texteditor;bundle-version="[3.16.600,4.0.0)",
  org.eclipse.e4.ui.css.swt.theme,
- org.eclipse.tools.templates.ui;bundle-version="[1.4.0,2.0.0)"
+ org.eclipse.tools.templates.ui;bundle-version="[1.4.0,2.0.0)",
+ org.apache.commons.io;bundle-version="[2.8.0,3.0.0)"
```

Comment:
Please prefer `Import-Package` for 3rd party libs, it simplifies releng work for SimRel